PyInstaller/hooks/hook-django.py

```diff
@@ -20,7 +20,9 @@
 
 logger = logging.getLogger(__name__)
 
-datas, binaries, hiddenimports = hooks.collect_all('django')
+# Collect everything. Some submodules of django are not importable without
+# considerable external setup. Ignore the errors they raise.
```

Comment:
Should this be re-wrapped? Or perhaps split so that each sentence is in its line?